Components/StatusMapping/OrderTransactionMapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace MollieShopware\Components\StatusMapping;
+
+
+use MollieShopware\Components\Constants\PaymentStatus;
+use Shopware\Models\Order\Status;
+
+class OrderTransactionMapper
+{
+    /**
+     * @param string $status
+     * @return array
+     */
+    public static function mapStatus($status)
+    {
+        $targetState = null;
```

Comment:
the outer scope had an exception on a non-existing and undefined mapping
could we just change that to a nullable return, and throw the exception in here instead of providing an array return,
that would be the most transparent one
otherwise please change to a class instead of array

Reply:
Moved return to own Struct class